manifests/manage/bzr.pp

```diff
@@ -0,0 +1,18 @@
+# @summary Manage the Bazaar source code manager package
+#
+# @param package_name
+#   name of package to manage
+#
+# @param package_ensure
+#   ensure state of the package resource
+#
+# @example
```

Comment:
please give the example a name, that makes the REFERENCE.md more readable:
```suggestion
# @example simple include
```

Reply:
done